assets/config/install/install.php

```diff
@@ -487,6 +489,16 @@ function goBack() {window.history.back()}
 					<input name=\"gmlevel\" type=\"text\" maxlength=\"10\" class='form-control' id=\"gmAccess\" placeholder=\"3\" value=\"3\" required/>
 					<small id=\"vpColumnHelp\" class=\"form-text text-muted\">What level GM should be allowed to access the GM panel?</small>
 				</div>
+				<hr/>
+				<div class=\"form-group\">
+					<label for=\"hashAlgorithm\">Password Hashing Algorithm</label>
+					<select name=\"hash-algorithm\" class='form-control'>
+						<option value=\"bcrypt\">BCrypt</option>
+						<option value=\"sha1\">SHA1</option>
+						<option value=\"sha512\">SHA512</option>
+					</select>
+					<small id=\"hashAlgorithmHelp\" class=\"form-text text-muted\">What algorithm should be used to hash passwords?</small>
```

Comment:
Some more text on which to select would be nice here, but not needed.

Reply:
i'm not sure what else to put honestly. if the user doesn't know what this means by what's already there, then let's just make a suggestion for them. you happy with that?